pkcsutil.py

```diff
@@ -43,7 +43,7 @@ def export_to_pkcs12(filename, key, cert, passphrase=None):
     parser = argparse.ArgumentParser(description='pkcsutil converts a PEM formatted certificate/private key to PKCS12.')
     parser.add_argument('--key',  help='Private key filename, PEM encoded.')
     parser.add_argument('--cert', help='Certificate filename, PEM encoded.')
-    parser.add_argument('--passphrase', help='Passphrase for the private key.', default=None)
+    parser.add_argument('--passphrase', help='Passphrase for the private key.', default="")
```

Comment:
please use the convention of single quotes for string literals. 

```suggestion
    parser.add_argument('--passphrase', help='Passphrase for the private key.', default='')
```